manifests/config.pp

```diff
@@ -22,15 +21,72 @@
     mode    => '0600',
   }
 
-  file {'/etc/httpd/conf.d/pulp.conf':
+  apache::vhost { 'pulp-http':
+    priority            => '05',
+    docroot             => '/srv/pulp',
+    port                => 80,
+    servername          => [$::fqdn],
+    serveraliases       => [$::hostname],
+    additional_includes => '/etc/pulp/vhosts80/*.conf',
+    add_default_charset => 'UTF-8',
+  }
+
+  apache::vhost {'pulp-https':
```

Comment:
WSGISocketPrefix is set from apache::mod::wsgi class, so I didn't thought it's necessary to set it explicitly.